packages/commonwealth/client/scripts/controllers/chain/substrate/democracy_referendum.ts

```diff
@@ -332,11 +327,6 @@ export class SubstrateDemocracyReferendum extends Proposal<
       });
     if (democracyProposal) return democracyProposal;
 
-    const collectiveProposal = chain.council?.store.getAll().find((p) => {
```

Comment:
A little worried about this -- what happens in the case where it _is_ a collective proposal?